elastalert/elastalert.py

```diff
@@ -1384,15 +1397,9 @@ def get_alert_body(self, match, rule, alert_sent, alert_time, alert_exception=No
         return body
 
     def writeback(self, doc_type, body):
-
+        writeback_index = self.writeback_index
         if(self.is_atleastsix()):
-            writeback_index = doc_type
-            if writeback_index == 'silence':
-                writeback_index = 'elastalert_silence'
-            elif writeback_index == 'past_elastalert':
-                writeback_index = 'elastalert_past'
-        else:
-            writeback_index = self.writeback_index
+            writeback_index = self.get_six_index(doc_type)
```

Comment:
instead of doing this local variable assignment at "runtime", can you do this once earlier upstream and just update the canonical `self.writeback_index`?

Reply:
It's different based on what doc_type you want to upload, so there is no canonical index. ``self.writeback_index`` is the prefix.